.gitignore

```diff
@@ -98,3 +99,6 @@ pip-selfcheck.json
 
 # Offline version file
 src/mlx/warnings/__version__.py
+
+# plantuml generation
+src/*.plantuml
```

Comment:
What does this do?

Reply:
I generated a UML diagram with `pyreverse mlx.warnings -o plantuml`